src/database/duckdb/schema.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS tags (
+  height UBIGINT NOT NULL,
+  id BLOB NOT NULL,
+  tag_index USMALLINT NOT NULL,
+  created_at UBIGINT NOT NULL,
+  tag_name BLOB NOT NULL,
+  tag_value BLOB NOT NULL,
+  is_data_item BOOLEAN NOT NULL
```

Comment:
@djwhitt 

I'm using UBIGINT for `height` and `created_at` so we don't need to worry about this numbers in the future.

For `tag_index` I'm using USMALLINT, but I think we will have more than 65k data items/bundle in a near future, right? Maybe we should change that to a UINT?

Reply:
I believe `tag_index` is the index of the tag within the data item as opposed to within the bundle.

Reply:
You're right 🤦